styles/app.scss

```diff
@@ -1,5 +1,6 @@
 $icon-font-path: "~bootstrap-sass/assets/fonts/bootstrap/";
 
+@import "../node_modules/pace/themes/blue/pace-theme-flash.css";
```

Comment:
Is the file included directly in the file ? Or this is compiled to a css `@import` that trigger another http request ?